scripts/check-release-startup.sh

```diff
@@ -15,9 +15,9 @@ sup() {
 script() {
     sup crossbar_maintenance create_account 'compte_maitre' 'royaume' 'superduperuser' 'pwd!'
     sleep 3
-    sup kazoo_perf_maintenance json_metrics | python -m json.tool
+#    sup kazoo_perf_maintenance json_metrics | python -m json.tool
     sleep 1
-    sup kazoo_perf_maintenance graphite_metrics 'compte_maitre' 'clu1' 'royaume'
+#    sup kazoo_perf_maintenance graphite_metrics 'compte_maitre' 'clu1' 'royaume'
```

Comment:
Log clutter for no benefit afaict. Was going to ask tomorrow about why they're here.